apis/builder/validators.yaml

```
@@ -0,0 +1,37 @@
+post:
+  tags:
+    - Builder
+  operationId: "registerValidator"
+  summary: Register or update a validator's block building preferences.
+  description: |
+    Registers a validator's preferred fee recipient and gas limit.
+
+    Builders should verify that `pubkey` corresponds to an active or pending
+    validator, and that `signature` is valid under `pubkey`. Otherwise, builder
+    must error. Requests with `timestamp` less than or equal to the previous
+    announcement must error. Requests with `timestamp` greater than 10 seconds
+    +/- the builder's local time must error.
```

```suggestion
    announcement must error. Requests with `timestamp` greater than 10 seconds
    in the future must error.
```
Old messages must still be valid, for instance if a new builder comes online.

---

Good point - updated.